HaulageApplication/HaulageTests/DriverRecordTripExpensesTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Haulage.BaseClasses.BillingHandler;
+using Haulage.DatabaseExecutionServices;
+using Haulage.viewModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Haulage;
+
+namespace HaulageTests
+{
+    [CollectionDefinition("DriverRecordTripExpensesTests", DisableParallelization = true)]
```

Comment:
this should be something like "DatabaseTests" as seen in other classes

Reply:
Now changed